mautic/mautic
# fix static config file in tests

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | maybe
| New feature/enhancement? (use the a.x branch)      | no
| Deprecations?                          | no
| BC breaks? (use the c.x branch)        | no
| Automated tests included?              | yes
| Issue(s) addressed                     | Fixes an issue with automated tests if the config file is not in the default location

#### Steps to test this PR:

1. Checkout the code locally 
2. Run automated tests for this
  - `composer test -- --filter ConfigControllerFunctionalTest` or
  - `ddev exec composer test -- --filter ConfigControllerFunctionalTest`

(I guess the automated tests that run with each PR should also show if there is a problem with this)

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10810"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
